### ISSUE TITLE
Org Graph Produces Almost Blank Graph

### ISSUE THREAD

**Issue by jrbrodie77** [org-roam/org-roam#1422]
#### Steps to Reproduce
Go to one of my .org files in my roam dir.  Call org-roam-graph.

I get a graph, but all I see is a small grey rectangle in the middle.  Can you recommend steps to debug?

- Org-roam commit: https://github.com/jethrokuan/org-roam/commit/commithashhere

**Comment by nickstares** [comment]
I'm having the same issue.

**Comment by jrbrodie77** [comment]
I did some investigation.  When I checked *messages* after calling org-roam-graph, there is a .dot file shown like this:

Wrote /var/folders/sg/wf7nvv452f960xykn6510x1h0000gn/T/graph.QIXy9D.dot

I can manually create a graph by running a command like:

dot -Tpng -O graph.QIXy9D.dot

or

dot -Tsvg -O graph.QIXy9D.dot

The problem seems to be with the reader of the .svg.  If I open the .svg with Chrome, it looks great.  If i open it with Emacs, I get a blank grey square.  So the problem is either with the .svg file produced by dot or the reader that Emacs uses for .svg.

*Anyone have any ideas on how to debug Emacs SVG opening?*
